### ISSUE TITLE
Breastfeeding assessment issues for YI

### ISSUE THREAD

**Issue by kaledip** [WorldHealthOrganization/smart-emcare-cmr#16]
The element "Difficulty Breastfeeding Observed" is a derived element. It needs to be calculated.
Other elements in breastfeeding assessment need to e reviewed for conditionality, not appearing correctly.

**Comment by magp18** [comment]
I have that one as readonly so you see the calculated value (yes or no) but you can't change it. I see that Iraq has it as "hidden" so it will be calculated in the background and not be shown to the user, should I do it like that as well?

**Comment by kaledip** [comment]
Yes we can implement it like Iraq,just the element used for calculation are different in Cameroon

**Comment by magp18** [comment]
This is done as per 0.0.129
